README.md

```diff
@@ -189,27 +191,105 @@ You can read and modify the configurations in `config-*.ini` files, accordingly:
+  - `ScreenshotsDirectory`: Configures the folder dedicated to storing all the images (for example the heatmap reports) generated by the processor. We recommend set this folder in a mounted directory (such as */repo/data/processor/static/screenshots*)
+  - `DashboardURL`: Sets the url where the frontend is running. Unless you are using a custom domain, you should keep this value as https://beta.lanthorn.ai/.
+  - `EnableSlackNotifications`: A boolean parameter to enable/disable the Slack integration for notifications and daily reports. We recommend not editing this parameter directly and manage it from the [UI](https://beta.lanthorn.ai) to configure your workspace correclty.
+  - `SlackChannel`: Configures the slack channel used by the notifications. The slack channel chosen must exist in the workspace configured.
+
+- `[Api]`
+  - `Host`: Configures the host IP of the processor's API (inside docker). We recommend don't change that value and keep it as *0.0.0.0*.
+  - `Post`: Configures the port of the processor's API (inside docker). Take care that if you change the default value (*8000*) you will need to change the startup command to expose the configured endpoint.
+  - `SSLEnabled`: A boolean parameter to enable/disable https/ssl in the API. We recommend setting this value in *True*. 
+  - `SSLCertificateFile`: Specifies the location of the SSL certificate (required when you have *SSL enabled*). If you generate it following the steps defined in this Readme you should put */repo/certs/<your_ip>.crt*
+  - [`SSLKeyFile`]: Specifies the location of the SSL key file (required when you have *SSL enabled*). If you generate it following the steps defined in this Readme you should put */repo/certs/<your_ip>.crt*
+
+- `[Core]`:
+  - `Host`: Sets the host IP of the *QueueManager* (inside docker).
+  - `QueuePort`: Sets the port of the *QueueManager* (inside docker).
+  - `QueueAuthKey`: Configures the auth key required to interact with the *QueueManager*.
+
+- `[Area_N]`:
+
+  A single processor can manage multiple areas and all of them must be configured in the config file. You can generate this configuration in 3 different ways: directly in the config file, using the [UI](https://beta.lanthorn.ai) or using the API.
+  - `Id`: A string parameter to identify each area. This value must be *unique*.
+  - `Name`: A string parameter to name each area. Although you can repeat the same name in multiple areas, we recommend don't do that.
+  - `Cameras`: Configures the cameras (using the *ids*) included in the area. If you are configuring multiple cameras you should write the ids separated by commas. Each area should have unless one camera.
+  - `NotifyEveryMinutes` and `ViolationThreshold`: Defines the *period of time* and *number of social distancing violations* desired to send notifications. For example, if you want to notify when *occurs more than 10 violations every 15 minutes*, you must set `NotifyEveryMinutes` in 15 and `ViolationThreshold` in 10.
```

Comment:
```suggestion
  - `NotifyEveryMinutes` and `ViolationThreshold`: Defines the *period of time* and *number of social distancing violations* desired to send notifications. For example, if you want to notify when *more than 10 violations happen in 15 minutes*, you must set `NotifyEveryMinutes` in 15 and `ViolationThreshold` in 10.
```

We wanted to move away from the `violations` term and use `close contact` instead. But I think social distancing violations is clear